.gitignore

```diff
@@ -20,3 +20,4 @@ examples/tensorflow/distributed_mnist/data
 doc/_build
 **/.DS_Store
 venv/
+*~
```

Comment:
Can this make it into the commit message as well.

Reply:
Yeah I accidentally added an emacs file on our fork. You mean you want a new commit in this PR with a message about that change?

Reply:
It's a good addition, happy for it to be part of this PR. If we leave it in this PR, can the commit message just reference the addition. Something like

"Upload files to S3 using multipart uploads.

Emacs temporary files covered in .gitignore"

Reply:
Yeah I'm happy to add it. Any idea if amending my commit message and force pushing to our fork will do it?

Reply:
Yeah, should be fine

Reply:
Ok that worked